deepchem/data/datasets.py

```diff
@@ -876,7 +896,7 @@ def __iter__(self):
     return TorchDataset()
 
   @staticmethod
-  def from_DiskDataset(ds):
+  def from_DiskDataset(ds: "DiskDataset") -> "NumpyDataset":
```

Comment:
Why is this "DiskDataset" instead of `DiskDataset` directly? Sorry if this is a newbie question about mypy!

Reply:
This is a workaround for the fact that Python doesn't allow forward references.  We can't refer to DiskDataset here, because it hasn't been defined yet at this point in the file.  The workaround is that you put it in quotes, and mypy evaluate the string.

Reply:
Ah that makes sense!